.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-22.04]
+        os: [ubuntu-latest]
```

Comment:
All this reproducibility work, and we backslid here... not that GitHub keeps those old images around forever.  Maybe it doesn't matter. :shrug:

Reply:
I had to bump us to 22.04 last week to save us from brew doom. Don't like changing defaults, which is why I rolled it back now we're on Nix. /shrug

Maybe what we should do is set `ubuntu-22` instead of `ubuntu-latest` as the default in sbt-typelevel, and bump it from there.

Reply:
It's probably not worth fussing over.  There are still more moving parts, like actions not being pinned to particular commits.  What's here is enough to provide a batteries-included development environment and stabilize the things most practically likely to change beneath us.